### PR TITLE
Entity Fields Delete Test

### PR DESCRIPTION
**Pull request by sergeonevdah**
refactor
split test
https://trello.com/c/0apbPcwr/212-entity-fields-delete-tc

### CODE REVIEW

**Review by VictoriaRogo — APPROVED** [review]

File: src/test/java/EntityFieldsDeleteTest.java [hunk 1]
```diff
@@ -76,4 +79,76 @@ public void createNewFields() {
...
+        WebElement resultCommentsElement = driver.findElement(By.xpath("//div[contains(text(),'" + comments + "')]"));
+        WebElement resultNumberElement = driver.findElement(By.xpath("//div[contains(text(),'" + number + "')]"));
+        WebElement resultDecimalElement = driver.findElement(By.xpath("//div[contains(text(),'" + decimal + "')]"));
+
+        Assert.assertEquals(resultTitleElement.getText(), title);
+        Assert.assertEquals(resultCommentsElement.getText(), comments);
+    }
+
+    @Test(dependsOnMethods = "createNewFields1")
+    public void deleteFields () {
```
Comment (Stan575) [inline src/test/java/EntityFieldsDeleteTest.java:123]: This test at the end doesn't have any verification that a record has been deleted, can you add one?

File: src/test/java/EntityFieldsDeleteTest.java [hunk 2]
```diff
@@ -76,4 +79,76 @@ public void createNewFields() {
...
+        WebDriverWait wait = new WebDriverWait(driver, 3);
+
+        WebElement goFields = driver.findElement(By.xpath("//li[@id='pa-menu-item-45']/a"));
+        goFields.click();
+
+        WebElement recycleBinElement = driver.findElement(By.xpath("//i[contains(text(),'delete_outline')]"));
+        ProjectUtils.click(driver, recycleBinElement);
+
+        WebElement currentTitleDeleteElement = driver.findElement(By.xpath("//b[text()='" + title + "']"));
+        wait.until(ExpectedConditions.elementToBeClickable(currentTitleDeleteElement));
```
Comment (Stan575) [inline src/test/java/EntityFieldsDeleteTest.java:150]: You may not need this explicit wait unless there's special case, element is already found in the line above

File: src/test/java/EntityFieldsDeleteTest.java (hunk 1 above)
Comment (sergeonevdah) [reply src/test/java/EntityFieldsDeleteTest.java:123]: This is a test only for created, next step > next test

File: src/test/java/EntityFieldsDeleteTest.java (hunk 2 above)
Comment (sergeonevdah) [reply src/test/java/EntityFieldsDeleteTest.java:150]: just in case

File: src/test/java/EntityFieldsDeleteTest.java (hunk 1 above)
Comment (Stan575) [reply src/test/java/EntityFieldsDeleteTest.java:123]: Then it's not a test, you can make it a function and use it in next test

Comment (sergeonevdah) [reply src/test/java/EntityFieldsDeleteTest.java:123]: I'm still working on it, while I decided to leave it like this

**Review by Stan575 — APPROVED** [review]

**Pull request merged**